tests/unit/core/components/TestSurveyNotice.js

```diff
@@ -32,6 +40,12 @@ describe(__filename, () => {
     return shallowUntilTarget(<SurveyNotice {...props} />, SurveyNoticeBase);
   };
 
+  const getNoticeProp = (root, prop) => {
+    const notice = root.find(Notice);
+    expect(notice).toHaveProp(prop);
+    return notice.props()[prop];
```

Comment:
Can we write `notice.prop(prop)` instead?

Reply:
Hah. Whoops.